integration-tests/ci-visibility/playwright-tests/todo-list-page-test.js

```diff
@@ -0,0 +1,13 @@
+const { test, expect } = require('@playwright/test')
+
+test.beforeEach(async ({ page }) => {
+  await page.goto(process.env.PW_BASE_URL)
+})
+
+test.describe('playwright', () => {
+  test('should work with failing tests', async ({ page }) => {
+    await expect(page.locator('.hello-world')).toHaveText([
+      'Hello Warld'
```

Comment:
"Warld"? If this is really a typo... it looks like the test is not checking it correctly.

Reply:
"should work with failing tests"
ok, is expected to fail, sorry

Reply:
indeed 😄